logitrail-woocommerce.php

```diff
@@ -1022,19 +1037,34 @@ function update_product() {
         }
     }
 
-    public static function  logitrail_shipping_enabled($product_id) {
-        $product = wc_get_product( $product_id );
-        // Disable shipping on some product types
-        if ( in_array($product->get_type(), array("bundle")) ) {
+    /**
+     * @param WC_Product $product
+     * @param bool $ignore_bundled
+     * @return bool
+     */
+    public function logitrail_is_virtual($product, $ignore_bundled = false) {
+        $test = wc_pb_get_bundled_product_map($product);
+        // We don't want to ship virtual products, but bundled products are virtual so exclude them
```

Comment:
Doesn't this just tell is the product virtual or not? It doesn't make any decision about excluding anything?